README.md

```diff
@@ -1 +1,127 @@
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+---
+
+- **[User will have access to all the projects and used technologies]**
+- **[User will be updated with the recent working updates]**
+- **[User will be able to contact and purchase merchandise]**
+
+---
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+By following below steps a new developer can make instant use of the project.
+
+---
+
+Get a local copy up and running now:
+
+### Prerequisites
+
+In order to run this project you need:
+
+<h5>Make sure you have the following installed:</h5>
+
+- <strong>text editor</strong>: eg. vs-code, atom
+- <strong>node v16.x</strong>
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd your-folder
+  git clone https://github.com/FredrickOtabil/Personal_Portfolio.git
+  cd Personal_Portfolio
+```
+
+### Install
+
+Install this project with:
+
+Run:
+
+```sh
+  npm install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+- <div><p>Open the <span style="color: #002053; font-weight: 300">index.html</span> file with your browser</p></div>
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+ lhci autorun --upload.target=temporary-public-storage --collect.staticDistDir=.
+ npx hint .
+ npx stylelint "**/*.{css,scss}"
+```
+
+## 👥 Author <a name="authors"></a>
+
+👤 **Fredrick Mensah Otabil**
+
+- GitHub: [@githubhandle](https://github.com/FredrickOtabil)
+- Twitter: [@twitterhandle](https://twitter.com/ClarkCoder)
+- Linkedin: [@twitterhandle](https://linkedin.com/in/fredrick-otabil-6106371aa/)
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [Place registration form and merchandise basket] **[Upgrade the hmtl file]**
+- [Total of three projects on discussions] **[Create a description of main and side projects]**
+- [Include pre-Covid working experience] **[Update on latest working experiences]**
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to come in touch with the author and arrange a meeting and make your contributions truly valuable.
+
+## ⭐️ Show your support <a name="support"></a>
+
+As this is one of the author's first initiatives, please be gracious and show your support by DMing on Twitter or contributing to this project. All of your efforts are greatly appreciated and welcomed!
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+Many hours have been spent on this project. As a result, I'd like to thank my entire family for their support throughout this long journey, as well as the microverse team and my partners.
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+- **[Who will benefit from such a tool at the end of the day?]**
+
+  - [New Micronauts]
+
+- **[From where do I begin my journey with your Company?]**
+
+  - [It all starts with the filling of our registration form]
```

Comment:
- [x] The MIT license gives users express permission to reuse code for any purpose, sometimes even if the code is part of proprietary software. As long as users include the original copy of the MIT license in their distribution, they can make any changes or modifications to the code to suit their own needs. However, for this particular one, the MIT link is missing and therefore other developers can't be able to access the terms and conditions. Kindly ensure to fix this 👍